Model/Source/DefaultOptions.php

```diff
@@ -1,31 +1,43 @@
 <?php
 /**
  * All functions to handle insurance
- *
  * If you want to add improvements, please create a fork in our GitHub:
- * https://github.com/myparcelbe
+ * https://github.com/myparcelnl
  *
- * @author      Reindert Vetter <info@sendmyparcel.be>
- * @copyright   2010-2019 MyParcel
+ * @author      Reindert Vetter <info@myparcel.nl>
  * @license     http://creativecommons.org/licenses/by-nc-nd/3.0/nl/deed.en_US  CC BY-NC-ND 3.0 NL
- * @link        https://github.com/myparcelbe/magento
+ * @link        https://github.com/myparcelnl/magento
+ * @copyright   2010-2019 MyParcel
  * @since       File available since Release v0.1.0
  */
 
 namespace MyParcelBE\Magento\Model\Source;
 
+use BadMethodCallException;
 use Magento\Sales\Model\Order;
 use MyParcelBE\Magento\Helper\Checkout;
 use MyParcelBE\Magento\Helper\Data;
-use MyParcelBE\Magento\Services\Normalizer\ConsignmentNormalizer;
-use MyParcelBE\Magento\Model\Sales\Package;
 use MyParcelBE\Magento\Model\Sales\Repository\PackageRepository;
+use MyParcelNL\Sdk\src\Factory\DeliveryOptionsAdapterFactory;
+use MyParcelNL\Sdk\src\Model\Carrier\AbstractCarrier;
+use MyParcelNL\Sdk\src\Model\Carrier\CarrierFactory;
+use MyParcelNL\Sdk\src\Model\Carrier\CarrierPostNL;
+use MyParcelNL\Sdk\src\Model\Carrier\CarrierBpost;
 use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
 
 class DefaultOptions
 {
     // Maximum characters length of company name.
-    const COMPANY_NAME_MAX_LENGTH = 50;
+    private const COMPANY_NAME_MAX_LENGTH    = 50;
+    private const INSURANCE_BELGIUM          = 'insurance_belgium_custom';
```

Comment:
Ik denk dat deze constante, en de `INSURANCE_BELGIUM_AMOUNT` eronder een indicatie zijn dat dit nog niet klopt, weet niet of ze bij het instellen van verzekeringen ook voor problemen zorgen maar ik verwacht het wel. <- we horen het dan van de tester :-D